examples/tree-view/package.json

```diff
@@ -1,29 +1,33 @@
 {
-  "name": "tree-view",
-  "version": "0.0.1",
+  "name": "redux-toolkit-tree-view",
```

Comment:
:hand: Let's leave this as `"redux-tree-view"` - I don't think it's necessary to specify the "toolkit" part